Community/alumni.md

```diff
@@ -2,33 +2,61 @@
+|[Ria Gupta](#Ria-Gupta)|2020|GSoC|Social Currency Metric System|Valerio Cosentino, Samantha Venia, Logan|
+|[Tianyi Zhou](#Tianyi-Zhou)|2020|GSoC|Large Social Network Analysis and Anomaly Detection with Augur|Sean Goggins, Jonahz, Gabe Heim|
+|[Sarit Adhikari](#Sarit-Adhikari)|2020|GSoC|Machine Learning for Anomaly Detection in Open Source Communities|Sean Goggins, Carter Landis, Gabe Heim|
+|[Abhinav Bajpai](#Abhinav-Bajpai)|2020|GSoC|Implementing GitLab Data Collection Worker and Mapper to bind the responses of GitLab API, Github API & the Augur schema|Sean Goggins, Carter Landis|
+|[bistaastha](#bistaastha)|2020|GSoC|Build Workflow process for CHAOSS D&I Badging Project|Matt Snell|
+|[vchrombie](#vchrombie)|2020|GSoC|Creating Quality models using GrimoireLab and CHAOSS metrics|Valerio Cosentino, Aniruddha Karajgi|
+|[Saicharan Reddy](#Saicharan-Reddy)|2020|GSoC|Implementation of GitLab Data Collection Worker & Test Coverage Improvement|Elita Nelson, Sean Goggins, Jonahz|
+|[Akshara P](#Akshara-P)|2020|GSoC|Machine Learning for Anomaly Detection in Open Source Communities|Elita Nelson, Sean Goggins, Gabe Heim|
+|[Pratik Mishra](#Pratik-Mishra)|2020|GSoC|Machine Learning for Anomaly Detection in Open Source Communities|Elita Nelson, Sean Goggins, Gabe Heim|
+|[Ore-Aruwaji Oloruntola](#Ore-Aruwaji-Oloruntola)|2020|Outreachy|Build Workflow Process for CHAOSS Diversity & Inclusion Badging|Matt Germonprez, Matt Snell, Saleh Abdel Motaal|
+|[Parth Sharma](#parth-sharma)|2019|GSoC|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins|
+|[Bingwen Ma](#bingwen-ma)|2019|GSoC|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins, Matt Germonprez|
+|[Aniruddha Karajgi](#aniruddha-karajgi)|2019|GSoC|Implementing CHAOSS Metrics with Perceval|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Nishchith K Shetty](#nishchith-k-shetty)|2019|GSoC|Support of Source Code Related Metrics|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Keanu Nichols](#keanu-nichols)|2018|GSoC|Reporting of CHAOSS Metrics|Sean Goggins, Jesus Gonzalez-Barahona|
+|[Pranjal Aswani](#pranjal-aswani)|2018|GSoC|Reporting of CHAOSS Metrics: Refactoring the existing code and extending the capabilities of the Manuscripts Project|Valerio Cosentino, Jesus Gonzalez-Barahona|
 
 
 ## Mentees
 
+### Jaskirat Singh
+
+Jaskirat Singh selected for GSoD 2020
+
+#### Project Title
+
+Create a CHAOSS Community-wide Handbook
+
+
+#### Project Description
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam faucibus sapien nisl, sit amet semper urna imperdiet sit amet. Mauris tempor, dui eu placerat dapibus, sapien mi maximus purus, in porta nibh ante et turpis. Donec ac vestibulum enim. Donec ut iaculis lacus. Phasellus consectetur elementum vulputate. Donec pulvinar lectus sit amet condimentum sollicitudin. Vestibulum sit amet interdum tellus. Quisque vitae varius nibh, id lobortis quam.

```

Comment:
You can add the project description from here, https://developers.google.com/season-of-docs/docs/2020/participants/project-linuxfoundation-jaskiratsingh2000

You don't need the whole thing. How about just the initial paragraphs?